assets/javascripts/discourse/admin-discourse-ai-plugin-route-map.js

```diff
@@ -4,11 +4,9 @@ export default {
   path: "/plugins",
 
   map() {
-    this.route("discourse-ai", { path: "/" }, function () {
```

Comment:
This top level route was the problem because it clashed with `/admin/plugins/:id` for the root show route.